lib/util/index.coffee

```diff
@@ -187,7 +187,13 @@ convertExpandToObject = (expandOption) ->
 	else if isString(expandOption)
 		return "#{expandOption}": {}
 	else if isArray(expandOption)
-		return fromPairs(expandOption.map((key) -> [key, {}]))
+		return fromPairs(expandOption.map((value) ->
```

Comment:
NIT: since `value[key]` seems weird, would it make sense to rename `value` to (let's say) `option`?